assets/tests/components/inputModal.test.tsx

```diff
@@ -1,24 +1,25 @@
 import { jest, describe, test, expect } from "@jest/globals"
 import React from "react"
-import { render, fireEvent } from "@testing-library/react"
+import { render, waitFor } from "@testing-library/react"
 import InputModal from "../../src/components/inputModal"
 import { StateDispatchProvider } from "../../src/contexts/stateDispatchContext"
 import { initialState, closeInputModal } from "../../src/state"
+import userEvent from "@testing-library/user-event"
 
 describe("InputModal", () => {
-  test("can be dismissed with escape key", () => {
+  test("can be dismissed with escape key", async () => {
```

Comment:
Non-blocking nit: (I actually prefer for this to be addressed in a follow-up PR, so _very_ non-blocking) This test asserts something that we don't care about, which is that a particular function gets called. If it can be done without too much trouble, I'd rather have the test assert that the modal disappears from the screen.

... part of why I would rather have that be a follow-up is that this is a good change, and my suggestion ☝️ is the sort of thing that... usually is pretty easy, but occasionally winds up sending one down a horrible rabbit hole, and I don't want this PR blocked on said rabbit hole.